cmd/cleaner/main.go

```
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"log"
+	"time"
+
+	"github.com/hashicorp/consul/api"
+	"gopkg.in/Clever/kayvee-go.v2"
+)
+
+func main() {
+
+	log.Printf(kayvee.FormatLog("IP Cleaner", kayvee.Info, "Invoked", nil))
+	client, _ := api.NewClient(api.DefaultConfig())
```

Also, we should not ignore this error.